apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -934,14 +934,44 @@
             }
         },
         "fork_management" : {
-            "type"    : "object",
+            "type" : "object",
             "additionalProperties" : false,
             "properties" : {
                 "network_id" : {
                     "description" :
                     "Identification of the network in case of hard forks.",
                     "type" : "string",
                     "default" : "ae_mainnet"
+                },
+                "fork" : {
+                    "type" : "object",
+                    "additionalProperties" : false,
+                    "properties" : {
+                        "signalling_start_height" : {
+                            "description" : "Height at which signalling period starts.",
+                            "type" : "integer"
+                        },
+                        "signalling_end_height" : {
+                            "description" : "Height at which signalling period ends.",
+                            "type" : "integer"
+                        },
+                        "signalling_block_count" : {
+                            "description" : "Count of the key blocks in signalling period necessary for the fork to take effect.",
+                            "type" : "integer"
+                        },
+                        "fork_height" : {
+                            "description" : "Height at which the fork should occur.",
+                            "type" : "integer"
+                        },
+                        "info_field" : {
+                            "description" : "Pseudorandom number (part of key block) used by the miners indicating which fork they support.",
+                            "type" : "integer"
```

Comment:
🤔 

Is there any other portion of the user experience where this value is exposed? If so we could use that (`cb_...`?).

Though number seems simpler... I like it. May I suggest limiting it to between 0 and the max that can be expressed in the info field?

Reply:
Let's try using the number (at least for now), it looks simpler. Added `maximum` in the schema.